src/Main.php

```diff
@@ -0,0 +1,212 @@
+	}
+
+	public function add_assets() {
+		// Does this look like an attendee screen?
+		if ( empty( $_GET['page'] ) || 'tickets-attendees' !== $_GET['page'] ) {
+			return;
+		}
+
+		// Execute once only
+		if ( $this->assets_enqueued++ ) {
+			return;
+		}
+
+		$this->assets();
+	}
+
+	protected function assets() {
+		wp_enqueue_style(
+			'attendee-data-editor-styles',
+			$this->plugin_url . 'src/resources/css/attendee-data-editor.css'
+		);
+
+		wp_enqueue_script(
+			'attendee-data-editor-script',
+			$this->plugin_url . 'src/resources/js/attendee-data-editor.js',
+			[ 'jquery' ],
+			false,
+			true
+		);
+
+		wp_localize_script(
+			'attendee-data-editor-script',
+			'attendee_data_editor', [
+				'check' => wp_create_nonce( 'attendee-data-editor' ),
+				'save_btn' => esc_html__( 'Save!', 'tribe-ext-attendee-data-editor' ),
+				'uh_oh_on_save' => esc_html__( 'Something went wrong: your attendee data could not be saved.', 'tribe-ext-attendee-data-editor' ),
+				'success_on_save' => esc_html__( 'Successful update!', 'tribe-ext-attendee-data-editor' ),
+			]
+		);
+	}
+
+	public function add_edit_link( $value, $item, $column ) {
+		if ( 'meta_details' !== $column ) {
+			return $value;
+		}
+
+		$event_id    = absint( $item['event_id'] );
+		$product_id  = absint( $item['product_id'] );
+		$attendee_id = absint( $item['qr_ticket_id'] );
+
+		if ( ! $attendee_id || ! $event_id || ! $product_id ) {
+			return $value;
+		}
+
+		$label = esc_html__( 'Edit details', 'tribe-ext-attendee-data-editor' );
+		$link  = " 
+			<div> <a
+				class='edit-attendee-data' 
+				data-attendee-id='$attendee_id'
+				data-event-id='$event_id'
+				data-product-id='$product_id'
+				href='#'>$label</a> 
+ 			</div> 
+ 		";
+
+		return $link . $value;
+	}
+
+	/**
+	 * Get the editor HTML and send back to the browser.
+	 *
+	 * This repurposes the same view used for this purpose on the frontend.
+	 */
+	public function editor_content() {
+		if ( ! $this->editor_content_checks() ) {
+			wp_send_json_error();
+		}
+
+		$attendee_data = $this->get_attendee_array();
+		if ( ! $attendee_data ) {
+			wp_send_json_error();
+		}
+
+		ob_start();
+		Tickets_View::instance()->output_attendee_meta( $attendee_data );
+
+		wp_send_json_success( [
+			'html' => ob_get_clean()
+		] );
+	}
+
+	/**
+	 * Looks like a valid request to edit the attendee data?
+	 *
+	 * @return bool
+	 */
+	protected function editor_content_checks() {
+		return (
+			! empty( $_POST['event_id'] )
+			&& ! empty( $_POST['attendee_id'] )
+			&& wp_verify_nonce( $_POST['check'], 'attendee-data-editor' )
+		);
+	}
+
+	/**
+	 * Obtain the attendee array expected and needed to build the edit-attendee-data
+	 * form.
+	 *
+	 * This isn't particularly efficient, but the ET/ET+ API doesn't seem to expose
+	 * any better ways at present.
+	 *
+	 * @return null|array
+	 */
+	protected function get_attendee_array() {
```

Comment:
@barryhughes

**The only true bug (all other feedback is just suggestions):** In my testing, this works for WooCommerce but not RSVP tickets.